### PR TITLE
feat: add deeper file search (in subfolders)

### PR DESCRIPTION
**Pull request by ytvnr**
**Issue**

https://graviteecommunity.atlassian.net/browse/APIM-65

**Description**

Add capability to retrieve subscription schema form under schemas/subscriptions.

As for getSchema (which is looking in schemas folder), the first file is returned.

---
**Gravitee.io Automatic Deployment**

🚀 A prerelease version of this package has been published on Gravitee's private artifactory, you can:
 - use it directly by updating your project with version: `1.25.0-apim-65-api-with-webhook-SNAPSHOT`
 - download it from Artifactory [here](https://odbxikk7vo-artifactory.services.clever-cloud.com/gravitee-snapshots/io/gravitee/plugin/gravitee-plugin/1.25.0-apim-65-api-with-webhook-SNAPSHOT/gravitee-plugin-1.25.0-apim-65-api-with-webhook-SNAPSHOT.zip)

### CODE REVIEW

File: gravitee-plugin-core/src/main/java/io/gravitee/plugin/core/api/AbstractConfigurablePluginManager.java [hunk 1]
```diff
@@ -32,6 +33,9 @@
     extends AbstractPluginManager<T>
     implements ConfigurablePluginManager<T> {
 
+    public static final String DIRECTORY_SPLITTER_CHAR = "/";
+    private static final Pattern DIRECTORY_SPLITTER = Pattern.compile(DIRECTORY_SPLITTER_CHAR);
```
Comment (marcambier) [inline gravitee-plugin-core/src/main/java/io/gravitee/plugin/core/api/AbstractConfigurablePluginManager.java:37]: Should we remove that if it's unused ?

Comment (ytvnr) [reply gravitee-plugin-core/src/main/java/io/gravitee/plugin/core/api/AbstractConfigurablePluginManager.java:37]: goosd catch

**Review by marcambier — APPROVED** [review]

**Pull request merged**